_docs/_latest/configure/scanner/default-rules.md

```diff
@@ -52,3 +52,6 @@ resources.
 
 ## Service Account Key
   * User-managed service account keys should not be older than the date and time you specify.
+
+## Whitelist
```

Comment:
We should describe this as external project access.
```
## External Project Access
```

Reply:
modified as suggested